src/wrapper/objects/jlist.rs

```diff
@@ -77,8 +77,8 @@ impl<'a> JList<'a> {
 
         match result {
             Ok(val) => Ok(Some(val.l()?)),
-            Err(e) => match e.kind() {
-                &ErrorKind::NullPtr(_) => Ok(None),
+            Err(e) => match *e.kind() {
```

Comment:
New ergonomic style of Rust, seems to be no longer requires dereferencing?
Or is this not the case?

Reply:
Yep, works in recent versions, but I don't want to break backward compatibility. IMO it's not that important.